src/Tickets/Emails/Dispatcher.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Email Dispatcher class.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+/**
+ * Class Dispatcher.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Dispatcher {
+
+	/**
+	 * Stores the Email used in this Dispatcher.
+	 *
+	 * @since TBD
+	 *
+	 * @var Email_Abstract|null
+	 */
+	protected ?Email_Abstract $email;
+
+	/**
+	 * Sets the Email instance that will be used for this Dispatcher.
+	 *
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 *
+	 */
+	public function set_email( Email_Abstract $email ): void {
+		$this->email = $email;
+	}
+
+	/**
+	 * Gets the email instance for this Dispatcher.
+	 *
+	 * @since TBD
+	 *
+	 * @return Email_Abstract|null
+	 */
+	public function get_email(): ?Email_Abstract {
+		return $this->email;
+	}
+
+	/**
+	 * Send an email.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool success True if the email was sent.
```

Comment:
```suggestion
	 * @return bool Whether the email was sent successfully.
```
Copying the return of `wp_mail` as that covers both true and false.